ghc-src/Miso/Html/Internal.hs

```diff
@@ -84,6 +93,8 @@ instance L.ToHtml (VTree action) where
   toHtmlRaw = L.toHtml
   toHtml (VText x) | T.null x = L.toHtml (" " :: MisoString)
                    | otherwise = L.toHtml x
+  toHtml (VTextRaw x) | T.null x = L.toHtml (" " :: MisoString)
```

Comment:
I parroted the lines above, but I'm not sure if it's necessary.

Reply:
That's fine, this show is just for debugging purposes anyways.